packages/verification-vue-component/src/locales/it.json

```diff
@@ -65,12 +65,16 @@
                 "signedVia": "Firmato tramite",
                 "issuerAddress": "Chiave pubblica dell'emittente",
                 "documentHash": "Impronta numerica del documento",
+                "txHash": "Transaction hash",
                 "registrationDate": "Data e ora di emissione",
                 "registrationTransaction": "Transazione di emissione",
                 "revocationTransaction": "Transazione di annullamento",
                 "revocationDate": "Data e ora di annullamento",
                 "smartContractAddress": "Indirizzo del Smart Contract",
-                "signatureTransactions": "Transazioni di firma",
+                "signatureDetails": "Signature details",
+                "signatureLevel": "Signature level",
+                "jurisdiction": "Jurisdiction",
+                "downloadSignature": "Download signature",
```

Comment:
@ianaz could you please translate the changed labels to italian?

Reply:
Sure :) 
d4fe284

Reply:
Thanks 🙏